project.php

```diff
@@ -1840,52 +1836,50 @@ function do_smooth_reading()
                 "<b>$sr_deadline_str</b>"
             );
 
-            echo "<li>";
-            echo $sr_sentence;
-            echo "</li>\n";
+            echo "<p class='pa'>", $sr_sentence, "</p>";
```

Comment:
If you're putting this on one line, just embed the variable into the string directly:
```
echo "<p class='pa'>$sr_sentence</p>";
```